### ISSUE TITLE
Allow more control over resuming

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#1725]
The new Samsung resuming feature should have the ability to be enabled/disabled, whether that is combined with the existing Resume setting or not could be up for discussion, I think it would make sense to just have the one option.

Mentioned at https://www.universalmediaserver.com/forum/viewtopic.php?p=38307#p38307

@drakulis  thoughts?

**Comment by drakulis** [comment]
Yep, I agree. One "resume option seems to be enough. And it should be very easy to implement.
